sw/airborne/firmwares/fixedwing/main_ap.c

```diff
@@ -425,6 +425,10 @@ static inline void telecommand_task(void)
 
     /** Pitch is bounded between [-AUTO1_MAX_PITCH;AUTO1_MAX_PITCH] */
     h_ctl_pitch_setpoint = FLOAT_OF_PPRZ(fbw_state->channels[RADIO_PITCH], 0., AUTO1_MAX_PITCH);
+#if H_CTL_YAW_LOOP
+    /** Yaw is bounded between [-AUTO1_MAX_YAW_RATE;AUTO1_MAX_YAW_RATE] */
+    h_ctl_yaw_rate_setpoint = 10 * FLOAT_OF_PPRZ(fbw_state->channels[RADIO_YAW], 0., AUTO1_MAX_YAW_RATE);
```

Comment:
since it is a rate and not an angle like the other ones it was just to low to make a difference. so I peeked into the rate loop of stabilization_attitude.c (line 353)
thats working much better - never the less 10 might not be the final solution but I really don't want to make it configurable.